playbook/spec/pb_kits/playbook/pb_currency/currency_spec.rb

```diff
@@ -13,6 +13,7 @@
   it { is_expected.to define_enum_prop(:size).with_default("md").with_values("sm", "md", "lg") }
   it { is_expected.to define_prop(:symbol).with_default("$").of_type(Playbook::Props::String) }
   it { is_expected.to define_enum_prop(:variant).with_default("default").with_values("default", "light", "bold") }
+  it { is_expected.to define_prop(:abbreviate).of_type(Playbook::Props::Boolean) }
```

Comment:
Maybe will worth it to test the default as well in case someone change it in future.

Reply:
I'll try that too

Reply:
done